index.css

```diff
@@ -0,0 +1,757 @@
+  background-repeat: no-repeat;
+  width: 150px;
+  height: 150px;
+  object-fit: cover;
+  padding-left: 5%;
+  padding-top: 5%;
+  margin-left: 14px;
+}
+
+.guest img:hover,
+.guest img:active {
+  box-shadow: 2px 2px 2px 2px rgb(0 0 0 / 20%);
+}
+
+#guest3 {
+  display: none;
+}
+
+#guest4 {
+  display: none;
+}
+
+#guest5 {
+  display: none;
+}
+
+#guest6 {
+  display: none;
+}
+
+.guest {
+  display: flex;
+  margin-top: 44px;
+}
+
+.ftsp-txt {
+  margin-left: 10px;
+  margin-top: 10px;
+}
+
+.ft-name {
+  font-size: 16px;
+}
+
+.p1 {
+  font-size: 10px;
+  font-style: italic;
+  color: #ef5a4a;
+}
+
+.p2 {
+  font-size: 10px;
+}
+
+.loadm-btn {
+  outline: none;
+  width: 93%;
+  border: 1px solid #e5e0e0;
+  margin-top: 46px;
+  margin-left: 11px;
+  margin-right: 11px;
+  margin-bottom: 15px;
+  height: 40px;
+}
+
+.loadm-btn:hover {
+  color: #fff;
+  background-color: #ff3f2a;
+  outline: none;
+}
+
+.loadm-btn:active {
+  color: #ef5a4a;
+  transform: scale(0.98);
+  font-weight: 600;
+}
+
+.loadm-btn:disabled {
+  background-color: #fff;
+  color: #ef5a4a;
+}
+
+.fttxt-div {
+  margin-left: 10px;
+  margin-top: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .darknav {
+    display: flex;
+    background-color: #3e3e3e;
+  }
+
+  .darkcont {
+    width: 85%;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .darkcont a {
+    margin-left: 16px;
+    color: #fff;
+  }
+
+  .menuBar {
+    display: none;
+  }
+
+  #desktop-nav-bar {
+    display: flex;
+    justify-content: center;
+    box-shadow: 0 0 0 2px #c7c0c0;
+    border: 1px solid #fff5f5;
+    height: 47px;
+    align-items: center;
+  }
+
+  .desktop-items {
+    display: flex;
+    height: 244px;
+    list-style: none;
+    margin-right: 5%;
+  }
+
+  .sf-btn {
+    border: 3px solid #ee7263;
+    font-size: 10px;
+    width: 80px;
+    height: 31px;
+    font-family: 'lato', sans-serif;
+    margin-top: -8px;
+    color: #f38a7d;
+  }
+
+  .head-para {
+    font-size: 35px;
+  }
+
+  .logo-div {
+    margin-top: 30px;
+    display: flex;
+    justify-content: center;
+  }
+
+  .logo-div img {
+    width: 40px;
+    height: 40px;
+    position: relative;
+    bottom: 4px;
+  }
+
+  .list-nav {
+    margin-top: 122px;
+  }
+
+  .toplh {
+    font-size: 12px;
+    position: relative;
+    color: #353535;
+    top: 4px;
+    right: 2px;
+  }
+
+  .ftsp-txt {
+    width: 46%;
+  }
+
+  .topltxt {
+    font-size: 8px;
+    position: relative;
+    color: #353535;
+    width: 71px;
+    right: -1px;
+  }
+
+  .top-logo {
+    margin-top: 84px;
+  }
+
+  .home-txt {
+    display: flex;
+    height: 479px;
+    margin-left: 20%;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .nav {
+    color: #3e3e3e;
+    font-size: 11px;
+    margin-left: 60px;
+  }
+
+  .nav:hover {
+    text-decoration: none;
+  }
+
+  .navl:hover,
+  .navl:active {
+    text-decoration: underline;
+    color: #ee7263;
+  }
+
+  .sf-txt {
+    font-size: 70px;
+  }
+
```

Comment:
- [ ] Hey @Sonikak004 you have done well in implementing the design for this project. However, there are some adjustments I would suggest to improve the UI of your project. As part of the project requirements, 
- The pages should look almost identical to the original design. 
- The pages should have the exact same colors, for text, backgrounds, borders, etc, from the original design.

* Basic color swatch: #272A31, #EC5242, #D3D3D3

- The pages should use the same fonts or similar fonts from the original design.

* Fonts used originally: Cocogoose, Lato

> Your design
<img width="1117" alt="Screenshot 2023-04-27 at 14 28 55" src="https://user-images.githubusercontent.com/71140133/234884170-d6c1a3dd-9a2d-41d9-8ee6-25a2150c5f64.png">


> Template design
<img width="775" alt="Screenshot 2023-04-27 at 14 27 41" src="https://user-images.githubusercontent.com/71140133/234884562-8ce7a4c6-f01f-4394-827b-b66989a57a44.png">